kmq.js

```diff
@@ -36,6 +36,25 @@ client.on("message", (message) => {
     }
 });
 
+client.on("voiceStateUpdate", (oldMember, newMember) => {
```

Comment:
The parameters to the callback are `oldVoiceState` and `newVoiceState`, not specific to member